lib/usd/utils/DiffCore.cpp

```diff
@@ -13,14 +13,13 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 //
-#include "AL/usd/utils/DiffCore.h"
-#include "AL/usd/utils/SIMD.h"
+#include "DiffCore.h"
+#include "SIMD.h"
```

Comment:
Prefer absolute paths.

Reply:
As in here - https://github.com/Autodesk/maya-usd/pull/288#discussion_r381584257 ?

Reply:
@robthebloke 

I don't understand the significance behind absolute path inside the shared library. This is what I see has been done in your code base prior to this PR and believe is what you referring to:

1-  In the top level AL's CMakeLists.txt, USDUTILS_INCLUDE_LOCATION is set to usdutils sub-directory:
`set(USDUTILS_INCLUDE_LOCATION ${CMAKE_CURRENT_LIST_DIR}/usdutils)`
2- Inside usdutils directory we go 3 folders deep(AL/usd/utils) and in the CMakeLists.txt inside you add ${USDUTILS_INCLUDE_LOCATION} to target_include_directories
```
target_include_directories(${USDUTILS_LIBRARY_NAME} PUBLIC
    ${USDUTILS_INCLUDE_LOCATION} 
)
```
3- Hence, you can include things like so: #include "AL/usd/utils/DebugCodes.h"

AFAIK, there is no significant difference inside the shared library itself to go with either "AL/usd/utils/DebugCodes.h" or "DebugCodes.h". 

However, I think I understand what you mean by absolute path if one wants to use these header files outside and inclusions like this: #include "AL/usd/utils/ForwardDeclares.h" makes sense to me.

In this PR, I have already installed the headers file inside "usdUtils" directory and inclusion to those header files is done like so:
e.g
#include <usdUtils/ForwardDeclares.h>
#include <usdUtils/SIMD.h>
#include <usdUtils/ALHalf.h>

I am not happy with the usdUtils name. Perhaps we could change `usdUtils` to something like these:

```
#include <mayaUsd/usd/utils/ForwardDeclares.h>
or 
#include <lib/usd/utils/ForwardDeclares.h>
```